src/hooks/axiosHooks.tsx

```diff
@@ -1,15 +1,24 @@
 import { Person } from 'types/person';
 import { Task } from 'types/task';
 import { User } from 'types/user';
+import { Project } from 'types/project';
 import { useGetData, CacheKey, IDataLoader } from 'hooks/getData';
 import { api } from 'store/api';
 
 // Temp
 
 // Helper function for the hook below, not technically a hook
-export const getPersonById = async (id: number): Promise<Person> => {
-  const response = await api.get(`/personnel/${id}`);
-  return response.data;
+export const getPersonById = async (
+  personId: string
+): Promise<Person | null> => {
+  try {
+    const res = await api.get(`/personnel/${personId}`);
+    return res.data;
+  } catch (e) {
+    console.error(e);
+    // throw e;
```

Comment:
code comment

Reply:
Also is it reasonable to have a 404 on requesting a person?  Is it always an error of there's no person returned?